packages/gallery/src/components/item/videos/videoItem.js

```diff
@@ -269,8 +269,7 @@ class VideoItem extends GalleryComponent {
       baseClassName += ' playedOnce';
     }
     // eslint-disable-next-line no-unused-vars
-    const { marginLeft, marginTop, ...restOfDimensions } =
-      this.props.imageDimensions || {};
+    const imageDimensions = this.props.imageDimensions || {};
```

Comment:
In which case we don't have the `imageDimensions` prop?

Reply:
I don't think we have a case like that, so I guess this empty object Is useless

Reply:
it seems that i was wrong, we don't always have it at first

Reply:
at least with the unit tests (maybe they are not written well)